src/utils/vc.js

```diff
@@ -103,6 +105,34 @@ export function getDockRevIdFromCredential(credential) {
   return blake2AsHex(credential.id, RevEntryByteSize * 8);
 }
 
+/**
+ * Checks if a credential has a credentialStatus property and it has the properties we expect
+ * If status doesnt exist or is invalid, throws an error.
+ * @param credential
+ */
+export function verifyCredentialStatus(credential) {
+  if (credential.credentialStatus) {
```

Comment:
Isn't the issue raising concern about assuming credential has property `credentialStatus`? 
`status = jsonld.getValues(credential, 'credentialStatus')` is sufficient?

Reply:
Maybe not. This is the definition of `getValues`.
https://github.com/digitalbazaar/jsonld.js/blob/7cf2f7155f7d9af6b1c60404cbeeb87788694883/lib/util.js#L304
Not as powerful as we hoped.

Reply:
Maybe of expand (jsonld.expand) first and use the appropriate URI status